Extend/ListModel.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Extend;
+
+/**
+ * @inheritdoc
+ */
+class ListModel extends ListModel_parent
+{
+
+    /**
+     * Function for loading the list including inactive items
+     *
+     * @return array
+     */
+    public function getListIncludingInactive()
+    {
+        $oListObject = $this->getBaseObject();
+        $sFieldList = $oListObject->getSelectFields();
+        $sQuery = 'select ' . $sFieldList . ' from ' . $oListObject->getViewName();
+        $this->selectString($sQuery);
+
+        return $this;
```

Comment:
php doc is that you return an array, are you sure $this is an array? as it is a class.